README.md

```diff
@@ -0,0 +1 @@
+## This is my blog
```

Comment:
@danielstich Please take a look at this section in particular

Reply:
I'll take a look

Reply:
Looks good!